README.md

```diff
@@ -2,7 +2,8 @@
 
 WebView2Utilities help you develop and debug Microsoft Edge WebView2 apps.
 
-![Screenshot of WebView2 Utilities](screenshot.png)
+![image](https://github.com/david-risney/WebView2Utilities/assets/5773562/00846e42-6556-49c2-91ba-efffa45668ca)
```

Comment:
You changed the alt text from 'Screenshot of WebView2 Utilities' to just 'image'. Would be good to at least keep as 'Screenshot of WebView2 Utilities' if not improve the image description for the alt text =)